Dangerfile

```diff
@@ -35,7 +35,10 @@ tracks_checker.check_tracks_changes(
 
 view_changes_checker.check
 
-pr_size_checker.check_diff_size(max_size: 500)
+pr_size_checker.check_diff_size(
+  max_size: 300,
+  file_selector: ->(path) { !path.include?('Tests/') }
```

Comment:
Just to be sure, does that condition cover all Tests we have in the app? I.e. we have `WooCommerceTests` under the `WooCommerce` folder, the `YosemiteTests` under `Yosemite`, etc

Reply:
Yep, that was what I figured based on Woo's file structure.
I indeed noticed that there are several folders and files related to tests and this is a condition I found to capture them: it should ignore cover any file under a folder finishing with `Tests/`. Some examples:

- WooCommerce/WooCommerce**Tests/** UnitTests.xctestplan
- WooCommerce/WooCommerce**Tests/** POS/Mocks/MockCardPresentPaymentService.swift
- Networking/Networking**Tests/** Responses/account-username-suggestions.json
- Storage/Storage**Tests/** Mocks/MockFileManager.swift
- Hardware/Hardware**Tests/** PaymentIntentParametersTests.swift

Reply:
Awesome! That makes sense. Thanks, Ian